### ISSUE TITLE
Trying to generate an image results in an error.

### ISSUE THREAD

**Issue by MrXen0morph** [ebolam/KoboldAI#279]
ERROR      | __main__:text2img_horde:9229 - <!doctype html>
<html lang=en>
<title>404 Not Found</title>
<h1>Not Found</h1>
<p>The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.</p>

**Comment by ebolam** [comment]
Looks like we have two issues. The above looks like some connection issue to Horde. We also have a summarizer issue (logits processor adders for KoboldAI Models is affecting summarizer) that is preventing me from getting to what you have above.

**Comment by MrXen0morph** [comment]
Huh. This error was popping up like... 1-3 weeks ago too? Can't say for sure, but it was a thing, at least on my side, for quite a while.

**Comment by ebolam** [comment]
Fixed now
